File: dotnet-desktop-guide/framework/winforms/controls/snippets/how-to-iterate-through-all-nodes-of-a-windows-forms-treeview-control/cs/Student.cs
using System.Collections;

namespace project
{
    // The basic Student class.
    public class Student
    {
        protected ArrayList _studentSubjects = new ArrayList();

        public Student(string studentName)
        {
            StudentName = studentName;
        }

        public string StudentName { get; set; } = "";

        public ArrayList StudentSubjects => _studentSubjects;
    }
}

File: dotnet-desktop-guide/framework/winforms/controls/snippets/how-to-iterate-through-all-nodes-of-a-windows-forms-treeview-control/vb/Student.vb
Namespace project
    Public Class Student
        Protected _studentSubjects As ArrayList = New ArrayList()

        Public Sub New(studentName As String)
            Me.StudentName = studentName
        End Sub

        Public Property StudentName As String = ""

        Public ReadOnly Property StudentSubjects As ArrayList
            Get
                Return _studentSubjects
            End Get
        End Property
    End Class
End Namespace
